cmd/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/dilyara4949/employees-api/internal/controller"
+	"github.com/dilyara4949/employees-api/internal/domain"
+	"github.com/dilyara4949/employees-api/internal/repository/employee"
+	"github.com/dilyara4949/employees-api/internal/repository/position"
+	"github.com/dilyara4949/employees-api/internal/route"
+	"log"
+	"net/http"
+)
+
+func main() {
+	storageP := &domain.PositionsStorage{
```

Comment:
I think, what you meant to do is to have `storage` as a DB substitute, but in this case I can't see benefit of that as I mentioned in one of the comments. 

On a side note, even if we use initialization here, you should not put on type user task to correctly initialize your structure explicitly (such as exposing Storage field as public and expecting it to be initialized). Use constructors for those cases when creating an instance without parameters will not guarantee it's correct work.